feature/mypage/src/main/java/com/mument_android/mypage/MyPageActivity.kt

```diff
@@ -45,6 +48,35 @@ class MyPageActivity : BaseActivity<ActivityMyPageBinding>(R.layout.activity_my_
         }
     }
 
+    //각 웹뷰로 이동
+    private fun clickListenerWebView(){
+        with(binding){
+            //자주묻는질문
+            btnMyPageGoFAQ.setOnClickListener {
+                initIntent("https://www.naver.com/")
+            }
+            //문의하기
+            btnMyPageGoInquiry.setOnClickListener{
+                initIntent("https://www.naver.com/")
+            }
+            //앱정보
+            btnMyPageGoAppInfo.setOnClickListener{
+                initIntent("https://www.naver.com/")
+            }
+            //뮤멘트 소개
+            btnMyPageGoIntroduceMument.setOnClickListener{
+                initIntent("https://www.naver.com/")
+            }
+        }
+    }
+
+    private fun initIntent(url: String) {
+        val intent = Intent(this, WebViewActivity::class.java)
+        intent.putExtra("url", url)
+        ContextCompat.startActivity(this, intent, null)
```

Comment:
- 해당 방법으로 사용하면 됩니다!!
```suggestion
        startActivity(intent)
```

Reply:
오호라이~ 그냥 하는거였네요 ㅋㅎㅋㅎ 감사합니당~!